homeassistant/components/aws/notify.py

```diff
@@ -32,10 +32,10 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-async def get_available_regions(hass, service):
+async def get_available_regions(hass: HomeAssistant, service: str):
```

Comment:
Is it possible to add the return type? I think it's a list of strings.
```suggestion
async def get_available_regions(hass: HomeAssistant, service: str) -> list[str]:
```

Reply:
Indeed